CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contribution Guide #
+# Contribution Guide
```

Comment:
@zhexuany I do not mean to offend you. I searched all docs just now and found `#...#` is rarely used.

<img width="375" alt="screen shot 2017-05-25 at 13 40 02" src="https://cloud.githubusercontent.com/assets/8832222/26437432/c08655e4-414f-11e7-94b4-2d9becea8d83.png">

In my opinion, it's vital that keeping the same style for a project rather than the concrete syntax. It's no point to argue about which syntax is better, just follow the majority :)